modules/RTC/TraceablePeerConnection.js

```diff
@@ -969,16 +971,18 @@ TraceablePeerConnection.prototype._remoteTrackAdded = function(stream, track, tr
     const sourceName = this.signalingLayer.getTrackSourceName(trackSsrc);
     const peerMediaInfo = this.signalingLayer.getPeerMediaInfo(ownerEndpointId, mediaType, sourceName);
 
-    if (!peerMediaInfo) {
-        GlobalOnErrorHandler.callErrorHandler(
-            new Error(`${this}: no sourceInfo available for ${ownerEndpointId}:${sourceName} track creation failed!`));
+    // Assume default presence state for remote source. Presence can be received after source signaling. This shouldn't
+    // prevent the endpoint from creating a remote track for the source.
+    let muted = true;
+    let videoType = VideoType.CAMERA;
 
-        return;
```

Comment:
Could you please add more details why we were not creating the track before and why do we want to create it now?

Reply:
Before ssrc-rewriting, source signaling and presence were received on the same channel, i.e., on the strophe ws. So there is a little chance of an endpoint receiving those messages out of order. However, with ssrc-rewriting, the sender sends source signaling and presence on the same strophe ws but on the receiver, source info is received on the bridge ws and presence on the strophe ws. There is a higher probability for these messages to be received out of order.
Till now client always assumed that presence is received before source-signaling but a remote track can be created assuming default presence.